README.md

```diff
@@ -47,6 +47,8 @@ use serenity::model::channel::Message;
 use serenity::framework::standard::macros::{command, group};
 use serenity::framework::standard::{StandardFramework, CommandResult};
 
+use tokio;
+
```

Comment:
This isn't necessary. Since Rust 2018, crates get automatically imported to all modules in a crate.
```suggestion
```